packages/test/stochastic-test-utils/src/describeFuzz.ts

```diff
@@ -24,12 +39,21 @@ function createSuite<TArgs extends StressSuiteArguments>(
  */
 export interface StressSuiteArguments {
 	/**
-	 * Whether the current run is a stress run, which generally runs longer or more programatically generated tests.
-	 *
-	 * Packages which use this should generally have a `test:stress` script in their package.json for conveniently running
-	 * the equivalent checks locally.
+	 * It indicates the "stress level" of tests. A value of 0 means the test is not run in stress mode, while
+	 * higher values indicate increasingly stressful testing. This value also acts as a multiplier for the
+	 * test count and timeout threshold. A stress run generally takes longer time or more programatically generated
+	 * tests.
 	 */
-	isStress: boolean;
+	stressMode: StressMode | undefined;
+}
+
+/**
+ * @internal
+ */
+export enum StressMode {
+	Short,
```

Comment:
nit: probably nice for readability here to make the values of this the corresponding strings